frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.scss

```diff
@@ -60,7 +74,11 @@
           justify-content: center;
         }
       }
+      .form-group {
+        margin-bottom: 0px;
+      }
       .form-row {
+        margin-bottom: 1rem;
```

Comment:
Silly question, but I've always been confused with `rem` -  all my understanding of it is that it is relative to the font size. Just wondering what scenarios `rem `is most often used over `px ` etc

Reply:
We have some inconsistency in this area for sure. REM is indeed relative to the font size, so any area where spacing relative to the font size makes sense is where rems should be used (many areas).